Add clarification about using server side options (#136)
passkeydeveloper/passkeys.dev: content/en/docs/use-cases/bootstrapping/index.md

      const available = await PublicKeyCredential.isConditionalMediationAvailable();

      if (available) {
        try {

>           // 2.1) Retrieve authentication options for

Reviewer: Are these numbers supposed to be a reference to something? They read like spec references as-is, and if they're not I'm not sure why they use sub-numbering instead of just "1:", "2:", etc...

Author: This is indeed not yet very consistent, will remove. 👍

          // Retrieve authentication options for `navigator.credentials.get()`
          // from your server.
          const authOptions = await getAuthenticationOptions();
          // This call to `navigator.credentials.get()` is "set and forget."
          // The Promise will only resolve if the user successfully interacts
          // with the browser's autofill UI to select a passkey.
          const autoFillResponse = await navigator.credentials.get({
          const webAuthnResponse = await navigator.credentials.get({
            mediation: "conditional",
            publicKey: {
              ...authOptions,
              // see note about userVerification below
              userVerification: "preferred",
            }
          });
           // Send the response to your server for verification.

>            // Authenticate the user if the response is valid.

Reviewer: Why did this line get the axe? Is there a reason the RP should **not** log the user in after successful verification?

Reviewer: For clarification this line was in for those (like me) who prefer to read code, so I think I'd like to see this second line remain/get reworded if need be, and essentially get duplicated in the text around the code blocks for those who don't enjoy reading code. Does that make sense?

Author: All right, it's just there's a lot of duplication between text and code, code and text. I suppose this is something to clean up / align eventually. In the meantime, fully understand your point and will add it again. 👍

- If the user selects the "Passkey from another device" option (NOTE: the exact text will vary slightly by platform), then the browser/platform will guide the user through using a FIDO2 security key or the  Cross-Device Authentication (CDA) flow to use a passkey from their smartphone or tablet to deliver a WebAuthn response to the `navigator.credentials.get()` call.

> - Send the WebAuthn response to your server for verification and additional security checks.

Reviewer: I don't think this line goes far enough. "...and then do what?" It's implied the user should get logged in, but why not explicitly mention that for sake of thoroughness?

- If the user selects the "Passkey from another device" option (NOTE: the exact text will vary slightly by platform), then the browser/platform will guide the user through using a FIDO2 security key or the  Cross-Device Authentication (CDA) flow to use a passkey from their smartphone or tablet to deliver a WebAuthn response to the `navigator.credentials.get()` call.

> - Send the WebAuthn response to your server for verification and additional security checks. When all checks are successful: the user is authenticated. Next, continue to start an authenticated session for this user.

Reviewer: ```suggestion
- Send the WebAuthn response to your server for verification and additional security checks. If all checks succeed then start an authenticated session for this user.
```
